### DISCUSSION TITLE
Perubahan nomor kontak Klinik Pratama Laurentius

### DISCUSSION THREAD

**Discussion by drhandy** [kawalcovid19/wargabantuwarga.com#882] [General]
Mohon bantuannya untuk mengganti nomor kontak Klinik Pratama Laurentius dari 081394027604 ke 0222030444. Terima kasih.

**Comment by zainfathoni** [comment]
Klinik yang ini kan ya? Baik, akan kami proses perubahannya.

https://www.wargabantuwarga.com/provinces/jawa-barat/tempat-vaksin-lokasi-vaksin-bandung-klinik-pratama-santo-laurentius-081394027604
